databroker/core.py

```diff
@@ -794,40 +794,18 @@ class BlueskyRun(intake.catalog.Catalog):
     get_datum_pages : callable
         Expected signature ``get_datum_pages(resource_uid) -> generator``
         where ``generator`` yields Datum documents
+    get_filler : callable
+        Expected signature ``get_filler() -> event_model.Filler``
+    transforms : Dict[str, Chttps://gist.github.com/gwbischof/6efafc15b620166a1492a5a610bc0ebfallable]
+        A dict that maps any subset of the keys {start, stop, resource, descriptor}
+        to a function that accepts a document of the corresponding type and
+        returns it, potentially modified. This feature is for patching up
+        erroneous metadata. It is intended for quick, temporary fixes that
+        may later be applied permanently to the data at rest
+        (e.g via a database migration).
     **kwargs :
         Additional keyword arguments are passed through to the base class,
         Catalog.
-    handler_registry : dict, optional
```

Comment:
Note to reviewers: this isn't related to the code changes in this PR. It should have been done in #465 but we missed and just noticed it while updating this docstring.